src/pages/LabwareDetails.tsx

```diff
@@ -89,6 +103,8 @@ export default function LabwareDetails() {
 
                     <StripyCardDetail term={'Date of Creation'}>{labware.created}</StripyCardDetail>
 
+                    {samplesDetails()}
```

Comment:
Just to note, may be for future, it is better to  avoid declaring a component within another component in same file. In this case , it is not that complex , but it can create performance issues in complex components.